binnacle/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ description: A new Flutter project.
 version: "1.0.0+1"
 
 environment:
-  sdk: ">=1.0.0-dev.68.0 <3.0.0"
+  sdk: ">=2.1.0 <3.0.0"
```

Comment:
Are we scrapping the idea of holding our minimum version to 1.0.0, or does a dependency require a greater version?

Reply:
This was due to the algorithm library if I recall correctly.

Reply:
Understood.

Reply:
Ya dart 1.0 is pretty restrictive